data/hai/hai reveal 4 middle.txt

```diff
@@ -1324,7 +1340,10 @@ mission "Hai Reveal [A13] A New Elder"
 			label summary
 			`	The freshest of the group, Batu, seems to twig to something first. "There's going to be an ultimatum, isn't there? The fleet is a threat."`
 			`	"Yes," says Sayari. "It's going to be a very broad ultimatum though, with lots of room for the Republic to decide how it's addressed; and no one is shooting at anyone without talking first. Now, all of you except <last> are traveling with Elder Teeneep. I suggest you go and prepare immediately."`
-			`	She approaches you as they leave. "Things might be tense now, but I promise, things aren't as bad as they look. I know Teeneep, and she has the influential backing of another long-time advocate for sensible change elsewhere who was also jusy elected. They are remaining here to reinforce her position. Teeneep displaced that Elder who threatened her in the chambers. It's upset all the long-term power dynamics in the Council, but it's a much better Council overall now. You're to come with us too, and I'm told I'm required to travel with you, as insurance."`
+			`	As you begin to walk away, Sayari motions for you. She leads you away to have a private discussion.`
+			choice
+				`	"What is it, Sayari?"`
```

Comment:
Having another choice here might be good.
```suggestion
				`	"What is it, Sayari?"`
				`	(Follow her.)`
```

Reply:
You would already have followed her by the time you ask, "What is it, Sayari?"

I do need a second option, though. I will ponder it. Either that, or I'll put the paragraph back together.

Reply:
Okay. There are two options now. How do you like them?